products/ocp4/profiles/moderate-node.profile

```diff
@@ -38,9 +38,6 @@ description: |-
 
 # CM-6 CONFIGURATION SETTINGS
 # CM-6(1) CONFIGURATION SETTINGS | AUTOMATED CENTRAL MANAGEMENT / APPLICATION / VERIFICATION
-extends: cis-node
+extends: moderate-node-rev-4
 
 filter_rules: '"ocp4-node" in platforms or "ocp4-master-node" in platforms or "ocp4-node-on-sdn" in platforms or "ocp4-node-on-ovn" in platforms'
```

Comment:
I wonder if we still need the filter here since there is only extending

Reply:
That's a good question. We can check to see if filters are also extended.

Reply:
Ok - I was able to remove the filter from the extended profile, rebuild the data stream, then compare the old profile to the newly generated one, and the rule selections were the same.